src/utils/base.test.js

```diff
@@ -0,0 +1,221 @@
+  });
+
+  describe('italic', () => {
+    it('should assign italic to fontStyle when italic is true', () => {
+      _this.props.italic = true;
+      const res = fn();
+      expect(res.fontStyle).toEqual('italic');
+    });
+    it('should assign normal to fontStyle when italic is false', () => {
+      _this.props.italic = false;
+      const res = fn();
+      expect(res.fontStyle).toEqual('normal');
+    });
+  });
+
+  describe('fontWeight', () => {
+    it('should assign bold to fontWeight when bold is true', () => {
+      _this.props.bold = true;
+      const res = fn();
+      expect(res.fontWeight).toEqual('bold');
+    });
+
+    it('should assign normal to fontWeight when bold is false', () => {
+      _this.props.bold = false;
+      const res = fn();
+      expect(res.fontWeight).toEqual('normal');
+    });
+  });
+
+  describe('caps', () => {
+    it('should assign uppercase to textTransform when caps is true', () => {
+      _this.props.caps = true;
+      const res = fn();
+      expect(res.textTransform).toEqual('uppercase');
+    });
+
+    it('should assign none to textTransform when caps is false', () => {
+      _this.props.caps = false;
+      const res = fn();
+      expect(res.textTransform).toEqual('none');
+    });
+  });
+
+  describe('margin', () => {
+    it('should assign margin value to margin ', () => {
+      const marginValue = '30px';
+      _this.props.margin = marginValue;
+      const res = fn();
+      expect(res.margin).toEqual(marginValue);
+    });
+  });
+
+  describe('padding', () => {
+    it('should assign padding value to padding ', () => {
+      const paddingValue = '30px';
+      _this.props.padding = paddingValue;
+      const res = fn();
+      expect(res.padding).toEqual(paddingValue);
+    });
+  });
+
+  describe('textColor', () => {
+    it('should assign textColor value to color ', () => {
+      const textColorValue = 'blue';
+      _this.props.textColor = textColorValue;
+      _this.context.styles.colors = { color: textColorValue };
+      const res = fn();
+      expect(res.color).toEqual(textColorValue);
+    });
+
+    it('should assign color to default color pink', () => {
+      _this.props.textColor = _this.context.styles.colors.primary;
+      const res = fn();
+      expect(res.color).toEqual('pink');
+    });
+  });
+
+  describe('textFont', () => {
+    it('should assign textFont value to fontFamily', () => {
+      const textFontValue = 'Times New Roman';
+      _this.props.textFont = textFontValue;
+      _this.context.styles.fonts = { font: textFontValue };
+      const res = fn();
+      expect(res.fontFamily).toEqual(textFontValue);
+    });
+  });
+
+  describe('fontFamily', () => {
+    it('should assign fontFamily to default font Helvetica', () => {
+      _this.props.textFont = _this.context.styles.fonts.primary;
+      const res = fn();
+      expect(res.fontFamily).toEqual('Helvetica');
+    });
+  });
+
+  // Need to figure out how to test for textSize
```

Comment:
I find it helpful to prefix comments like this with `@TODO`. For example:

```
// @TODO add test for textSize
```

Makes comments like this easily searchable across the codebase!